### ISSUE TITLE
Throttle/Brake Question

### ISSUE THREAD

**Issue by ManieCR** [josh31416/mqtt-rc-car#1]
Hi Josh31416

Thanks for your work, I used your script to connect the PS3 controller with node red on the RPi.

I have a question though (If you dont mind me asking), my throttle and brake is the the x-axis or the right joystick, which isnt a problem but I would like to use R2 for that instead. So my question, is that even possible? Because I assume pygame thinks there's one of those oldschool joysticks connected to the Pi instead of a PS3 controller?

**Comment by josh31416** [comment]
Absolutely. It’s been a while since I developed this script, but I’m pretty sure I used L2 and R2. In fact, you can see I use only one axis for steering and two for braking and throttle. In addition, I specify the movement axis for steering to be 0, and for throttle and braking to be -1. I’m from Spain, maybe it’s a matter of PS3 controllers being different in other countries.

In any case, you can check for yourself which button corresponds to which pygame joystick axis. Just program a script that outputs the value of an axis and change it until you find the R2 axis. Same for L2. Maybe there’s a function that outputs all axis’s values at the same time…

Let me know if this helped you. Good luck!!
